solr/config/schema.xml

```diff
@@ -374,6 +380,55 @@
       </analyzer>
     </fieldtype>
 
+    <!-- Analyzed Text, English -->
+    <fieldtype name="text_en" class="solr.TextField" positionIncrementGap="10000" autoGeneratePhraseQueries="true">
+      <analyzer>
+        <tokenizer class="solr.WhitespaceTokenizerFactory" />
+        <filter class="solr.ICUFoldingFilterFactory"/>  <!-- NFKC, case folding, diacritics removed -->
+        <filter class="solr.SynonymFilterFactory" synonyms="synonyms.txt" ignoreCase="true" expand="false"/>
+        <filter class="solr.WordDelimiterFilterFactory"
```

Comment:
I noticed that we're using `WordDelimiterGraphFilterFactory` in content search.  `WordDelimiterFilterFactory` (used here) is deprecated.  Not sure if this will cause any issues currently, but was wondering if we should update exhibits to use the non-deprecated version.

Reply:
👍 it'll be easier now than later.